README.md

```diff
@@ -1,134 +1,65 @@
 
-## Monitoring build with eBPF
+### Built With
 
-The set of examples under `build_with_ebpf` are a few examples of how without the right preventive controls around your supply chain you will have to rely on detective controls. It is still good practice from a defense in depth approach to still apply monitoring like this on the builds to still detect anomalous behaviors or when your preventive controls fail for any reason.
+Platform:
+* [Kubernetes](http://k8s.io/)
+* [Tekton Pipelines](https://tekton.dev/)
+* [Tekton Chains](https://github.com/tektoncd/chains)
+* [Spire](https://spiffe.io/)
+* [Kyverno](https://kyverno.io/)
 
-### Threats
+Tooling:
+* [Cosign/Sget](https://github.com/sigstore/cosign)
+* [Crane](https://github.com/google/go-containerregistry)
+* [Make](https://www.gnu.org/software/make/)
 
-The threats these examples emulate are the following:
+## Getting Started
 
-* Injecting unknown build tools into a container - DONE
-* Injecting unknown source code into shared drive - NOT DONE
-* Build scripts attempting to call out to internet - NOT DONE
-* Approved build tools performing suspicious activities like injecting binaries into memory and executing directly - DONE
+The following describes how to set up a dev environment with the Secure Software Factory installed.
 
-### Pre-Requisites
+### Prerequisites
 
-In order to run these examples, you'll need a system with the following
+Required:
+* Make
+* Kubernetes cluster (if not using local Minikube)
 
-* [Rust & Cargo](https://doc.rust-lang.org/cargo/getting-started/installation.html)
-  * Ensure you have x86_64-unknown-linux-musl target installed for static linking
-    `rustup target add x86_64-unknown-linux-musl`
-* [Nix](https://nixos.org/guides/install-nix.html)
+Optional Tools:
+* Crane
 
-### Setup
+### Installation
 
-TODO: Publish artifacts of binaries + containers
+TODO: Put key makefile commands
 
-```bash
-cd build_with_ebpf/bad_cargo
-./build.sh
-```
 
-## Emulating Attacks
+## Usage
 
-### Build on Host
+See `/examples` for examples of the 
 
-Normal Build:
+<!-- ROADMAP -->
+## Roadmap
 
-```bash
-cd build_with_ebpf/real_project
-cargo build --release
-```
+TODO: Put roadmap
 
-Build that hijacks the source files:
 
-```bash
-cd build_with_ebpf/real_project
-../bad_cargo/target/release/bad_cargo_inputs build --release
-```
+<!-- CONTRIBUTING -->
+## Contributing
 
-Build that hijacks the output:
+TODO: Create CONTRIBUTING.MD
 
-```bash
-cd build_with_ebpf/real_project
-../bad_cargo/target/release/bad_cargo_outputs build --release
-```
+## License
 
-Once you have run any of the above you can test it via:
+See LICENSE.MD
 
-```bash
-./target/release/real_project
-```
 
-You should get a "Goodbye, World" output on the hijacked ones.
 
-### Build in Container
+<p align="right">(<a href="#top">back to top</a>)</p>
```

Comment:
A bit nitpicking, but a markdown anchor would do the trick rather than embedding HTML into the markdown.

Reply:
Oops. Yeah, I took it from a template. I'm going to fix that .